extensions/cache/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Why do we need a `META-INF/bean.xml`?

Reply:
It is required for the `CacheRepository` injection into `AugmentedCacheAnnotationInterceptor`. This class is currently an `@ApplicationScoped` bean, but this can be changed and the `beans.xml` file can be removed if that's a problem.